src/gengowatcher/ui_textual.py

```
@@ -145,6 +244,496 @@ def render(self):
+
+        # Build Rich formatted content
+        lines = [
+            f"[#7FB4CA]Interval:[/] {check_interval}s",
+            f"[#98bb6c]Min $:[/] ${float(min_reward):.2f}",
+            f"[#e0af68]Auto:[/] {'[#98bb6c]On[/]' if autoaccept_enabled else '[#f7768e]Off[/]'}",
+            f"[#957fb8]WS:[/] {'[#98bb6c]On[/]' if ws_enabled else '[#f7768e]Off[/]'}",
+        ]
+        try:
+            self.query_one("#config-content", Static).update("\n".join(lines))
+        except Exception:
+            pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)